dist/redhat/scylla.spec

```diff
@@ -54,7 +54,7 @@ Group:          Applications/Databases
 Summary:        The Scylla database server
 License:        AGPLv3
 URL:            http://www.scylladb.com/
-Requires:       %{product}-conf %{product}-python3
+Requires:       %{product}-conf %{product}-python3 %{product}-node-exporter
```

Comment:
Here too. Better make just the superpackage depend on it. It's what most people install anyway.